scss/_patterns_buttons.scss

```diff
@@ -92,16 +117,48 @@ $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
     @include vf-focus($color-focus-positive);
   }
 
+  %p-button--positive-dark {
+    $color-positive-dark: adjust-color($color-positive, $lightness: -10%);
```

Comment:
@lyubomir-popov This is just provisional darker version of positive color, I'll let you adjust it properly.